src/js.rs

```diff
@@ -772,12 +829,13 @@ pub mod test_runner {
         let runner = runner_tokens.to_file_string()?;
         runner_file.write_all(runner.as_bytes())?;
 
-        js_file.keep()?;
-        rust_file.keep()?;
-        library_file.keep()?;
-        let (_, p) = runner_file.keep()?;
+        //        js_file.keep()?;
+        //        rust_file.keep()?;
+        //        library_file.keep()?;
+        //        let (_, p) = runner_file.keep()?;
         let test = TestCases::new();
-        test.pass(p);
+        //        test.pass(p);
+        test.pass(runner_file.as_ref());
         Ok(())
```

Comment:
what's going on here?

Reply:
I accidentally committed the `keep` stuff from when I was debugging. Undid that, but I find it convenient so I kept it around as comments.